src/apiServices/AuthServiceNode.ts

```diff
@@ -0,0 +1,96 @@
+import { postRequest, getRequest } from '../http'
+
+import { loginUser } from '../auth/loginUser'
+
+import { getHeaders } from '../util/getHeaders'
+import { errorHandler } from '../util/errorHandler'
+import { API_URL_PRODUCTION } from '../util/constants'
+
+import { JwtTokensInterface } from '..'
+
+export class AuthServiceNode {
+
+    public ethAddress: string
+    public password: string
+    public jwtTokens: JwtTokensInterface = {
+        access: '',
+        refresh: ''
+    }
+
+    constructor(ethAddress?: string, password?: string) {
+        this.ethAddress = ethAddress as string
+        this.password = password as string
+    }
+
+    public async login(): Promise<JwtTokensInterface | undefined> {
+        try {
+            const result = await loginUser(this.ethAddress, this.password, `${API_URL_PRODUCTION}/auth/login`)
+            const tokens: JwtTokensInterface = result.data.jwtTokens
+            this.setCredentials(tokens)
+            console.log('You are logged in!')
+            console.log(`Access: ${tokens.access}`)
+            console.log(`Refresh: ${tokens.refresh}`)
+
+            return tokens
+        }
+        catch (err) {
+            errorHandler(err)
+        }
+    }
+
+    public async logout() {
+        if (this.isUserLoggedIn()) {
+            try {
+                const headers = getHeaders(this.jwtTokens.access)
```

Comment:
Like in the other service, we can probably set `this.headers` as a property of the class. (Set it in `setCredentials`)